pkg/admission/apibinding/apibinding_admission.go

```diff
@@ -96,6 +97,19 @@ func (o *apiBindingAdmission) Admit(ctx context.Context, a admission.Attributes,
 		apiBinding.Spec.Reference.Workspace.Path = cluster.Name.String()
 	}
 
+	// set labels
```

Comment:
nit: don't describe what the code is doing but why

Reply:
really, in one liner section-title like comments? They should say what on a high level. The why is for longer explanations.